File: Chnegyi/形狀/ConsoleApplication5/ConsoleApplication5/Triangle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApplication5
{

    class Triangle
    {   
        public int bottom ;
        public int height ;
        public void input() {
            Program main = new Program();
            Console.WriteLine("輸入底:");
            bottom = main.consoletoint();
            Console.WriteLine("輸入高");
            height = main.consoletoint(); 

        }
        public int getarea() {
            return bottom * height / 2;

        }
      
    }
    
        

}

File: LiNNN/物件村題目/src/面積2/Triangle.java
package 面積2;

public class Triangle extends Shape{
        
        @Override
        public void Area(){
                System.out.println("請輸入長、高");
                double length = input.nextDouble();
                double height = input.nextDouble();
                
                double area = length * height / 2;
                
                System.out.println("面積 = " + area);
                
        }
}
